### PR TITLE
Adding search scope and ms.service values

### PR DESCRIPTION
**Pull request by JasonWHowell**
Repairing metadata for ms.service for Redis cache.
Adding missing ms.service for my team areas. 

Adding a search scope so that Azure CLI searches are unified from the TOC filter.

Adding custom search scopes for big data services so that, and the service-specific searches also work across all docsets, and Azure docs all up can be searched.

### CODE REVIEW

**Comment by JasonWHowell** [comment]
Spot checking the files to make sure things are working. Looks good

view-source:https://review.docs.microsoft.com/en-us/cli/azure/redis?view=azure-cli-latest&branch=pr-en-us-1663
`<meta name="ms.service" content="cache">`
`<meta name="scope" content="Cache for Redis, Azure, Azure CLI" />`

https://review.docs.microsoft.com/en-us/cli/azure/kusto?view=azure-cli-latest&branch=pr-en-us-1663
`<meta name="scope" content="Kusto, Azure Data Explorer, Azure, Azure CLI" />`
`<meta name="ms.service" content="data-explorer">`

Files that are not in a specific search scope use the global search scope.
view-source:https://review.docs.microsoft.com/en-us/cli/azure/managedservices?view=azure-cli-latest&branch=pr-en-us-1663
`<meta name="scope" content="Azure, Azure CLI" />`

image

**Comment by JasonWHowell** [comment]
#sign-off @rloutlaw

**Comment by JasonWHowell** [comment]
@sdwheeler do you have rights in this repo by chance?

File: docfx.json [hunk 1]
```diff
@@ -203,14 +203,11 @@
       "brand": "azure",
       "author": "rloutlaw",
       "ms.author": "routlaw",
```
Comment (sdwheeler) [inline docfx.json]: manager is required metadata. I believe that Barb Kess it now the manager that owns this content.

**Comment by sdwheeler** [comment]
Manager metadata requirement has been removed per: https://review.docs.microsoft.com/en-us/help/contribute/metadata-attributes?branch=master#manager-tag

**Pull request merged**